app/models/audit_trail.rb

```diff
@@ -0,0 +1,3 @@
+class AuditTrail
```

Comment:
This kinda falls into the "don't (re)test the framework/gem" 

I think it's covered by the smoke test from the acceptance standpoint. If this file wasn't working the smoke test would tell us immediately.